src/gettingstarted/cli.md

```diff
@@ -75,6 +75,55 @@ Options:
  --shell (-s)          Launch the shell
 ```
 
+### CLI features
+
+Additional settings to control the operation of the Platform.sh CLI can be managed in the configuration file (`.platform/local/project.yaml`) or environment variables. See the [`README` for the CLI for details](https://github.com/platformsh/platformsh-cli/blob/master/README.md#usage). 
+
+#### Auto-selecting your project
+
+When your shell's working directory is inside a local checkout of your project repository, the CLI will autodetect your project ID and environment, so you don't need to list them as parameters each time.
+
+In your home directory, for example, you need to provide the project ID as an argument each time:
+
+```bash
+~ $ platform project:info --project=acdefghijkl --environment=staging
+```
+You can instead get the same result with just:
+```bash
+~ $ cd myproject
+myproject $ project:info
+```
+
+You can also set a preferred project ID with the environment variables `PLATFORM_PROJECT`, `PLATFORM_BRANCH` and `PLATFORM_APPLICATION_NAME`.
+
+```bash
+export PLATFORM_PROJECT=acdefghijkl;
+export PLATFORM_BRANCH=staging;
+project:info
+```
+
+### Autocomplete on the command line
+
+Once installed, the `platform` CLI tool provides tab autocompletion for commands, options, and even some values (your projects, valid regions). eg:
+``` bash
```

Comment:
I'm not sure this renders well enough to be a benefit. Most people will understand what is meant by bash completion, anyway: https://pr-1123-j34rm2q-652soceglkw4u.eu-3.platformsh.site/gettingstarted/cli.html

Reply:
The illustration is not great, no, but I wanted to try and 'show it in action' because it's pretty dull to describe otherwise, but actually fun to watch.
In that format it does look cruddier that it did on my screen. I was wanting to do an animated gif, but decided against it this time.

Reply:
I'd say this is a good use of Asciinema if we want to include it.  Or we can just skip it, as I agree with Rob that anyone who is going to benefit from this feature already knows what it is.